docs/common/values/certificates.md

```diff
@@ -0,0 +1,88 @@
+# Certificates
+
+## Key: scaleCerts
+
+Info:
+
+- Type: `dict`
+- Default: `{}`
+- Helm Template:
+  - certPath: ✅
+  - keyPath: ✅
+
+Can be defined in:
+
+- `.Values`.scaleCerts
+
+---
+
+Before taking any action, it will check if middleware have populated
+the `ixCertificates` list, and that the `id` is included.
+
+For every item it will create a secret containing
+the certificate and the private key.
+
+Optionally, you can mount certificate, private key or both in the
+container as files.
+
+Examples:
+
+```yaml
+scaleCerts:
+  some_cert_name:
+    # ID Comes from the definitions on the GUI
+    id: 1
+    # Optional, Override Name
+    nameOverride: name_override
+    # Optional, If populated, it will mount the certificate in the container's path
+    certPath: /some/path/in/the/container/crt.key
+    # Optional, If populated, it will mount the private key in the container's path
+    keyPath: /some/path/in/the/container/key.key
+    # Optional, Allow the use of revoked certs, even if is not allowed globally
+    useRevoked: false
+    # Optional, Allow the use of expired certs, even if is not allowed globally
+    useExpired: false
+```
+
+---
+---
+
+## Key: scaleCertsList
```

Comment:
Both keys can be provided, but depending on the app, only one should be used.
eg. on `ix-chart` it might be better suited to expose the list style. as you can't know before hand what certs and how many a user might want to add.

But on ready made apps, the dict should be used and exposed.

I can add a check to only allow one of them to be defined.